components/ContentTemplates/ButtonsTemplate.tsx

```diff
@@ -467,6 +467,34 @@ export const ButtonsTemplate = () => {
 					</li>
 				</ul>
 			</TemplateSection>
+			<TemplateSection
+				sectionName="Touch Target Minimum"
+				title="Touch Target Minimum">
+				<p>
+					When it comes to creating accessible buttons, it's important that they
+					are easy to activate. To achieve this goal, WCAG suggests the
+					following:
+				</p>
+				<ul className="list">
+					<li>
+						Spacing: Avoid putting the buttons close together. Consider making
+						it at least 24 by 24 CSS pixels in size. It'll make it easier for
+						users who suffer from mobility impairments such as hand tremors or
+						are amputees to navigate mobile apps or websites.
+					</li>
+				</ul>
+				<p>
+					{" "}
+					If you're interested in learning more about this new standard, check
+					it out at{" "}
+					<a
+						href="https://www.w3.org/TR/WCAG22/#target-size-minimum"
+						className="blockLink">
+						{" "}
+						W3's website
+					</a>{" "}
+				</p>
```

Comment:
Links are already included in a section at the bottom of the page so this can be removed